hoomd/mpcd/RejectionVirtualParticleFiller.h

```diff
@@ -0,0 +1,173 @@
+
+#ifndef MPCD_REJECTION_VIRTUAL_PARTICLE_FILLER_H_
+#define MPCD_REJECTION_VIRTUAL_PARTICLE_FILLER_H_
+
+#ifdef NVCC
+#error This header cannot be compiled by nvcc
+#endif
+
+#include "VirtualParticleFiller.h"
+#include "hoomd/RandomNumbers.h"
+#include "hoomd/RNGIdentifiers.h"
+
+#include "hoomd/extern/pybind/include/pybind11/pybind11.h"
+
+namespace mpcd
+{
+
+//! Adds virtual particles to MPCD particle data for "sphere" geometry
+/*!
+ * <detailed description needed>
+*/
+
+template<class Geometry>
+class PYBIND11_EXPORT RejectionVirtualParticleFiller : public mpcd::VirtualParticleFiller
+    {
+    public:
+        //! Constructor
+        /*!
+         * \param sysdata - MPCD system data
+         * \param density - number density of MPCD solvent
+         * \param type - particle fill type
+         * \param T - Temperature
+         * \param seed - seed for PRNG
+         * \param geom - confinement geometry
+         */
+        RejectionVirtualParticleFiller(std::shared_ptr<mpcd::SystemData> sysdata,
+                                       Scalar density,
+                                       unsigned int type,
+                                       std::shared_ptr<::Variant> T,
+                                       unsigned int seed,
+                                       std::shared_ptr<const Geometry> geom)
+        : mpcd::VirtualParticleFiller(sysdata, density, type, T, seed), m_geom(geom), m_tmp_pos(0, m_exec_conf), m_tmp_vel_tag(0, m_exec_conf)
```

Comment:
I think this is sufficient:
```suggestion
        : mpcd::VirtualParticleFiller(sysdata, density, type, T, seed), m_geom(geom), m_tmp_pos(m_exec_conf), m_tmp_vel_tag(m_exec_conf)
```